src/ol/renderer/canvas/vectortilelayer.js

```diff
@@ -21,7 +21,7 @@ goog.require('ol.transform');
  * @const
  * @type {!Object.<string, Array.<ol.render.ReplayType>>}
  */
-ol.renderer.canvas.IMAGE_REPLAYS = {
+ol.renderer.canvas.VectorTileLayer.IMAGE_REPLAYS = {
```

Comment:
Looks like you'll want to assign this after `ol.renderer.canvas.VectorTileLayer` below.